ledger/ledger.go

```diff
@@ -387,7 +387,8 @@ func (v *Value) UnmarshalJSON(b []byte) error {
 }
 
 // Migration defines how to convert the given slice of input payloads into an slice of output payloads
-type Migration func(payloads []Payload) ([]Payload, error)
+// and returns the paths of the payload
+type Migration func(payloads []Payload, pathFinderVersion uint8) ([]Payload, []Path, error)
```

Comment:
Its not clear to me why the migration should return paths. I feel like if anything it should accept paths...